src/PolygonWorldID.sol

```diff
@@ -14,6 +14,22 @@ import {SemaphoreVerifier} from "semaphore/base/SemaphoreVerifier.sol";
 /// @dev This contract is deployed on Polygon PoS and is called by the StateBridge contract for each
 ///      new root insertion.
 contract PolygonWorldID is WorldIDBridge, FxBaseChildTunnel, Ownable {
+    /// @notice The selector of the `receiveRoot` function.
+    bytes4 receiveRootSelector;
+
+    /// @notice The selector of the `receiveRootHistoryExpiry` function.
+    bytes4 receiveRootHistoryExpirySelector;
```

Comment:
Add a `@dev` to each of these explaining why they're precomputed.

Reply:
done